src/main/java/core/basesyntax/reportservices/ReportToFileImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.reportservices;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.List;
+
+public class ReportToFileImpl implements ReportToFile {
+
+    @Override
```

Comment:
There should not be an empty line at the beginning of the method implementation.